### ISSUE TITLE
Blue Skies Research Based Tech Tree

### ISSUE THREAD

**Issue by pap1723** [KSP-RO/RP-0#667]
Some of you have been playing around with the tech tree that I made and have provided great feedback on it. As I have played through it, there have been some nice things and there have been some things that feel clunky with it. After some discussions with @NathanKell and @stratochief66 we discussed some ideas on how to improve things. I have put together some ideas and have the general guidelines for a tech tree.

Blue Skies Research is essentially research that does not have any tangible application. The goal is not to set out to research say a better solar panel, but it might be to research thermodynamics that then might possibly lead someone else into creating a better solar panel using some of the ideas from that research. Wikipedia describes it better: https://en.wikipedia.org/wiki/Blue_skies_research

@NathanKell had mentioned to me that Materials Science in a tech tree should be treated as a blue sky research node that doesn't really unlock anything, but instead unlocks the possibilities to unlock the next set of technologies from the research done.

@stratochief66 and @NathanKell both had the idea to break things out into smaller nodes. That means that a node shouldn't necessarily unlock 20 parts, but 4 nodes unlocking 5 parts is preferable. The player has more decisions to make and therefore options on how to suit their play-style or needs at that particular moment. Overall, this is going to have nodes that have much smaller science requirements to unlock, so extensive playtesting will be needed to see how the values need to be tweaked.

So the beginning of the tree will progress very similar to the current tree, but it will have more nodes worth less science points. This will allow the player to unlock some every couple of months instead of fast-forwarding 200 days to get the next node.

For the blue sky reasearches, I wanted to break things out into Eras of Spaceflight. Basically, the player will be able to research applied technologies for a period of years and then they will hit a stopping point for the current level of technology that they have. They will need to put some research into the next blue sky technology to progress to the next age. Here are the ages as I have currently thought about them in my head. Please let me know if you think some tweaking should be done (there should be, but I am stuck with it for now):

1. Post WWII Early Rocketry (1945-1955) - No Blue Sky Research required
2. Satellite Era (1956-1960)
3. Early Human Spaceflight (1961-1963)
4. Advanced Capsules & Interplanetary Probes (1964-1966)
5. Moon Exploration (1967-1971)
6. Space Stations & Deep Space Probes (1972-1980)
7. Spaceplanes & Exploration (1981-1985)
8. Long Term Space Habitation (1986-1995)
9. International Cooperation (1998-2008)
10. Commercial Space Age (2009-2017)
11. Near Future (2018-2049)
12. Colonization (2050 and Beyond)

Finally, here is a preview of the current tree as was in my head. The blue nodes are the normal nodes and the Green nodes are the Blue Sky Researches (I know, but it was late and I was working quickly). The orange are placeholders that mean I do not know if there should be a technology there yet. This is just the first 2 "Eras" and will obviously progress from here.

Pinging @leudaimon @rsparkyc @jwvanderbeck @PhineasFreak as all have expressed "some" interest before.

[Newest Tree Layout.pdf](https://github.com/KSP-RO/RP-0/files/1045143/Newest.Tree.Layout.pdf)

**Comment by leudaimon** [comment]
Very cool! Regarding general design, I think it's a very solid idea. Both increasing the number of nodes and the addition of the blue sky/paradigm shift idea are pretty interesting.
Some specific comments to the current nodes:

* Where do you intend to put the probe core and avionics parts currently in stability/early probes? Should be spread into the orange nodes between basic avionics and interplanetary probes, right? 

* Regarding solids, where would the Tiny Tim be placed? There is also the procedural solids, that I never got to use (usually don't even get the solids nodes), but I think could be present at least as soon as the baby sargeants.

* shouldn't basic capsules depend on EDL? Given they usally come with a heat shield built-in, the current design would allow capsules without the development of heat shields.

* Only the first 2 materials science nodes contain parts. Where would the other tank type and diamater upgrades be placed?

**Comment by PhineasFreak** [comment]
As i fail to see anything that is not awespme, i have a small request: please lock down the tech tree nodes and placement before starting populating it with parts.

**Comment by pap1723** [comment]
Phineas, why do you think that is a better option? So more people can place
quickly?

**Comment by PhineasFreak** [comment]
Yes. The current tech tree structure does not help you find where to place new parts (at least for the later nodes that are less fleshed out). At least that is my opinion.

**Comment by pap1723** [comment]
@PhineasFreak that makes sense. I will probably continue to add ideas of what should be in each node as I continue (as you see below some of them now).

**Comment by pap1723** [comment]
Hey @leudaimon thanks for the feedback and questions.

1) I have updated the original post with the newest version of the tree. There are additional nodes in the location that you mentioned.

2) For the Tiny Tim, I think it will either be available in the Start node like it is now, or with the first 1 point Post-War Rocketry Testing node. For the procedurals (which I never use) I do not know where they will be. I will defer to others that use them. Are they representative of the very earliest SRB's like the GCR Kick and Altair? I think they will be unlocked in the 1956 Solids node. After conversations with @NathanKell I have moved the Castor-1 unlock to that same node as it was the civilian version of the Sergeant SRM. With his changes to the tier levels and ton limitations on the buildings, I am not worried about players being able to abuse the early solids as I would be with a 40t limit as it currently stands.

3) Yes they should and now they do! Good call!

4) The Blue Sky Technologies in the Materials Science nodes will be the tank type and diameter upgrades for the most part. The other thing that might be added is applied technologies to the same path as the Blue Sky Techs. This all depends on play-through testing.

**Comment by NathanKell** [comment]
Tiny Tim was a WWII anti-shipping rocket (hence the jokey name, like Bazooka). It should stay in the start node IMO.

**Comment by leudaimon** [comment]
@NathanKell, Not that I disagree about the Tiny Tim... but that would apply to the A-4 and its guidance too, right? 
Of course, from a gameplay point of view, the A-4 is a huge upgrade that defined mostly all of the later liquid engine designs, both american and soviet, and it would make sense to have it as a technology developed with a space program in mind, and thus in the tech tree. 
I'm just picking on that to see if it's possible to come up with a good criterion for what to put in the starting node.

Btw, I didn't know a change in the buildings was being planned. Probably a good move, as you can do a lot with 40T (if you don't go R-7). Are those changes sketched anywhere?

**Comment by jwvanderbeck** [comment]
@leudaimon Well you could argue that there was time involved in **adapting** the V-2 into the A-4 for use on US rockets and use that as justification for pushing it up a node, rather than just being there at the start.

**Comment by NathanKell** [comment]
Yeah, it's the gameplay concern that's the thing. The only flimsy justification we can make is the work the US and Russians had to do to make A-4 work with sounding payloads and upper stages.

Perhaps a reasonable criterion might be: stuff used for civilian uses during the war or right at the end. That would allow the WAC and Tiny Tim but push the A-4 off a bit....? Or maybe the best criterion, sadly, really is "what's good for gameplay". It's not like the Bumper launches were after 1951 after all.

**Comment by NathanKell** [comment]
Ah, ninja'd by Agathorn.

**Comment by NathanKell** [comment]
Ah, here's another aspect to consider:
Like how KCT upgrade points just represent how large your staff is, what if science nodes also to some degree represent not what you _know_ but what _your program can do_. Science points are scientific-political capital, spent to improve your capabilities.

**Comment by leudaimon** [comment]
Good points you guys raise here. I think we have to accept gameplay justification, otherwise we end up too tight in historical reconstruction. Sorry if I looked picky, just looking after clarifying the proposal and design, which is fantastic already. The beauty of this new tree is there will be place for all those engine cfgs... 

Btw, in another look into the tree, early RTGs could be available at the Adv. Capsules and Interplanetary probes, couldn't them? They were not used extensively before Pioneer 10, but some experimental and low power versions were designed around that time (Did a quick lookup, apparently surveyor could have used them, and some transit satellites launched in 1961 had a puny 2.5W RTG).

**Comment by NathanKell** [comment]
No worries! :)

Yep, AIUI Transit was the first US use.

**Comment by pap1723** [comment]
@leudaimon @PhineasFreak 
I think this tree is rather complete (through Modern Day at least). I know there will be nodes that need to be added/moved/deleted etc, but I am going to work on getting a working copy in game. Let me know if you see any glaring issues.

[Newest Tree Layout.pdf](https://github.com/KSP-RO/RP-0/files/1053853/Newest.Tree.Layout.pdf)

**Comment by pap1723** [comment]
In-game layout of the tree. I think it works pretty well! There are some small issues with some of the dependency lines, but I don't think there is an easy way to re-arrange them to stop it from happening. As always, your feedback is appreciated!

http://i.imgur.com/fVMaBUP.png

**Comment by pap1723** [comment]
@NathanKell @PhineasFreak @jwvanderbeck @stratochief66 

Alright, the tree is completed and the parts are all placed. I have not added the Procedural Parts, RF Unlocks, RCS Unlocks, or Tank Unlocks to the tree yet. I will finish that this evening.

I want / need your opinions on what to do going forward for creating the tree. I know that in the past, everything was stored in Tree.yaml and was built by hand. What I have created with this spreadsheet will do much the same, but it requires some copy-paste functions. To me, the spreadsheet is vastly superior as it allows us to sort, edit, change, move and organize every part in RO. Using the current method, it is much harder to find if a specific part has been costed and placed.

That being said, I will defer to your preferences. I have to leave now, but I will fully explain the spreadsheet when I get home later this evening.

https://www.dropbox.com/s/6fhgeie182uptoh/Newest%20RP-0%20Tech%20Tree%20Parts%20-%20Copy.xlsm?dl=0

**Comment by NathanKell** [comment]
How recent a copy of Excel does it require? I'm back in 2003.

**Comment by pap1723** [comment]
It was written in 2013, but there are no formulas that will not work back
even earlier than that. If you have trouble opening it, I will save as an
older XLS version.

**Comment by NathanKell** [comment]
That'd be great! Actually, probably the best option is to do it in Google Docs, if we can. That way Linux folks can use it without touching grubby paid software, etc. :]
If it backports fine then Google Docs should be fine, the only issue I encountered was needing to convert tabs and linebreaks to char(n) statements.

**Comment by NathanKell** [comment]
I'll have to see about reusing the system for entry cost modifier generation too...since that's closely coupled to tree/costs anyway.

**Comment by PhineasFreak** [comment]
@pap1723 Now that you mention it, add the RCS PartUpgrades to the available UPGRADES list. Currently, the tech unlocks without showing up in the tree.

**Comment by leudaimon** [comment]
@pap1723 @NathanKell 
My first pass through the tree:

I will not discuss node costs by now. I think this will demand a lot of testing and balancing. However, looking at total science cost to get to some engines makes it look like prices are way too high.

Going by branches:

#### Science
Looks nice, with good variety. The only inconsistency I see is the aerobee science module. It could be in the starting node, given the science it gives is available there.

#### Avionics
Not looking into procedurals yet (I have some ideas for balancing, but this will have to wait for the non-procedurals anyway). Besides that, I think the sputnik can be pushed earlier to avionics prototypes, and the 20in core also be pushed to early avionics and probes.

#### Communications
Not much to say here. My only suggestion would be to have the communotron 16 and Reflectron KR-7 on lunar range communications. One thing I also think may be very nice is to extend the range of this Reflectron to be a GEO relay station for commanding vessels in LEO.

#### Energy
I think a tiny solar cell, vanguard like, is lacking in the game. How difficult would it be to reduce one of these models and make a part generating 3-4 W? Then, in the Satellite Era tier there could be two tech nodes, the first with this small solar cell, and then another with the panel already there (ST1)

#### Eletronics and Materials
It is not clear what exactly is unlocked on those nodes, besides the "eras". For materials I understand it's diameters and fuel types. What about eletronics?

#### Rocketry and staged combustion
Things are so different here it's hard to have an impression. The pace and order of unlocking of different rockets was seriously changed. I'm not saying this necessarily is a problem, but I think that in some cases, sticking with history might actually bring problems. For example, the RD-107/108 series coming before the Vanguard engine is very counter-intuitive, and does not help gameplay. For the staged combustion, I'm also surprised, because the S1.5400 and the Proton engines used to unlock together, and now there is a series of three empty nodes between them.
Some more specific rumbles: 
* The Taerobee aerobee and Tiny Tim booster are misplaced in Basic Rocketry, while their "stock" counterparts are in the starting node.
* The RD-100 series from RealEngines does not appear in the tree
* Why separation motors only in Basic rocketry?
* The AJ-10s were seriously pushed back, right? The early used to unlock with the RD-103, while the mid with the RD-107/8. Now the early unlocks with the RD-107/8, and the mid only with the H1.

#### Supersonics
Just one inconsistency, the X-1 cockpit is in the first node, while the conic cockpit that is a copy of it is on the third one, with the 1.25m cockpit.

**Comment by pap1723** [comment]
@leudaimon 

Thanks for the response and early feedback! I will go through each point to 

> I will not discuss node costs by now. I think this will demand a lot of testing and balancing. However, looking at total science cost to get to some engines makes it look like prices are way too high.

I agree with you. This is going to take extensive testing and right now I put the same costs for each node in the same Year.

> Science
Looks nice, with good variety. The only inconsistency I see is the aerobee science module. It could be in the starting node, given the science it gives is available there.

So the Aerobee science module from Taerobee I think is a variant that was later than the WAC-Corporal that is included in RO stock. However, that being said, I am going to move the Taerobee stuff all into the start node. It is not like it is better than the RO version, just different size diameters.

> Avionics
Not looking into procedurals yet (I have some ideas for balancing, but this will have to wait for the non-procedurals anyway). Besides that, I think the sputnik can be pushed earlier to avionics prototypes, and the 20in core also be pushed to early avionics and probes.

I could be convinced that you might be right, however, I want to see what the very early game feels like before making those kinds of changes to see how it all works and plays.

> Communications
Not much to say here. My only suggestion would be to have the communotron 16 and Reflectron KR-7 on lunar range communications. One thing I also think may be very nice is to extend the range of this Reflectron to be a GEO relay station for commanding vessels in LEO.

I would agree with you about the Communotron 16, but if a player (like @stratochief66) plays without RemoteTech, they would be without an antenna. The KR-7 (according to the description and speed at which it transmits data), it is a more advanced version of a dish than what was being thrown at the Moon in 1959. The Communotron 16 should be enough for that.

> Energy
I think a tiny solar cell, vanguard like, is lacking in the game. How difficult would it be to reduce one of these models and make a part generating 3-4 W? Then, in the Satellite Era tier there could be two tech nodes, the first with this small solar cell, and then another with the panel already there (ST1).

I use FASA so I have one, but I agree that a stock version of the solar panel should/could be included. All it takes is a simple CFG file that will shrink the ST1 in half and change the charge available.

> Electronics and Materials
It is not clear what exactly is unlocked on those nodes, besides the "eras". For materials I understand it's diameters and fuel types. What about eletronics?

Right now, absolutely nothing is unlocked in the Electronics nodes currently. That will need to be changed. Any ideas on what to unlock in there would be appreciated.

> Rocketry and staged combustion
Things are so different here it's hard to have an impression. The pace and order of unlocking of different rockets was seriously changed. I'm not saying this necessarily is a problem, but I think that in some cases, sticking with history might actually bring problems. For example, the RD-107/108 series coming before the Vanguard engine is very counter-intuitive, and does not help gameplay. For the staged combustion, I'm also surprised, because the S1.5400 and the Proton engines used to unlock together, and now there is a series of three empty nodes between them.

There are a lot of engine configs that are unlocked in nodes that might look blank right now. When we implement the PARTUPGRADE features on them, it will fill out the tree. As for the RD-107/108, @NathanKell and I talked about having the VERY early versions of the RD-107/108 available early, but for gameplay purposes, the Vanguard unlocking at the same time would make sense. The S1.5400 will be getting additional engine configs that will have 4 different versions that unlock in different tiers. The S1.5400 unlocked in 1960 for the Molniya launcher while the Proton didn't fly until 1966.

> Some more specific rumbles:
The Taerobee aerobee and Tiny Tim booster are misplaced in Basic Rocketry, while their "stock" counterparts are in the starting node.
The RD-100 series from RealEngines does not appear in the tree
Why separation motors only in Basic rocketry?
The AJ-10s were seriously pushed back, right? The early used to unlock with the RD-103, while the mid with the RD-107/8. Now the early unlocks with the RD-107/8, and the mid only with the H1.

As mentioned above, I will move the Aerobee stuff into the start node, just makes more sense.
The RD-100 from RealEngines is properly configured to unlock in the very first rocketry node, can you please confirm that is not showing up?

The AJ-10's were not pushed back so much as everything was pushed up. A lot of the earliest engines (RD-100-103) were pushed back too far in the previous tree due to the number of nodes available and wanting a progression.

> Supersonics
Just one inconsistency, the X-1 cockpit is in the first node, while the conic cockpit that is a copy of it is on the third one, with the 1.25m cockpit.

I will fix that, thanks!

**Comment by NathanKell** [comment]
Engine thoughts:

1. START: WAC, A-4, RD-100, sep motors, Tiny Tim
2. XASR-1, RD-101.
3. RD-102 (where else do we stick it?)
4. RD-103, NAA75-110 non-hydyne, AJ10-27, Navaho stuff - 1955
5. RD-107/8 PS1, LR43s [new], S-3 [new], X-405, RD-103M, NAA75-110 hydyne, AJ10-37 - 1956
6. missing RD-107/8 next step (I deleted this at some point), LR89/105-3, S-3D (i.e. slightly better RD107/8s, and the US engines we have now for Basic), RD-0105, AJ10-42 - 1957-8
7. RD-0109, Vostok/Luna RD107/8, US first upgrades. XLR81-BA-5. 1960
8. Molniya 107/8, RD-0107 (new config for 0110). Branch into Staged for S1.5400 first version. 1961

Not sure what to do with the AJ10s. There are lots of them. Agena, too: we know Agena A first launched in January 1959. We know AJ10-104 first launched in January 1960 (but per Gunter *not* the 104D, that came later). I'll trawl [Gunter's Thor page](http://space.skyrocket.de/doc_lau_fam/thor.htm) later.

Staged

1. S1.5400. ~1961
2. The upgrade to that (for better Molniya L), and the NK-9 and NK-9V ~1963. Or should the NK-9s be later?
3. Proton engines, Molniya M version of S1.5400. ~1965
4. NK-15, V. 1967?
etc.

Does that make sense?

For solids, what if we started the solid tree earlier, i.e. in line with liquid 2 and have the Aerojet Aerobee kick motor there, then a node in line with 5 for the Sergeant/Castor and Baby Sergeant and the original Vanguard kick motor (from GCR).

Apologies for not plotting that out earlier; it might require one more rocketry node?

Also, in terms of Balance (tm):
Since right now parts unlock immediately rather than taking time to unlock, the fact that the big engines cost more than the X-405 don't give them a time advantage. But the early big engines were _very_ unreliable, especially compared to the X-405, so I think that's how we need to balance things. We might also tweak their reliability curve _shapes_ too, so unlike before where you get 75% of the max reliability at 25% of the data, maybe it's linear for them, requiring far more flights and making them maybe not worth the cost (making skipping them, using X-405/NAA75-110/RD-103 for orbit, and saving up for second-gen big boosters a viable strategy).

**Comment by NathanKell** [comment]
I think we might want to skip a couple AJ10 configs, since some were nearly one-offs.

**Comment by ec429** [comment]
Things I've noticed so far:
- A-9 config on the A-4 engine is available straight away (Post-War Rocketry Testing), would it make more sense for it to be in Early Rocketry?
- The 4Mm antenna (Communotron 16; longAntenna) is in the Start node, should it be in Lunar Range Communications?  (I see @leudaimon already mentioned that.  Maybe you can do some MM magic to move it iff RT2 is installed?)
- Lack of sep motors early on is a pain, necessitating hot-staging.  Could you make a scaled-down version of the small sep motor that's available earlier, say in Early Rocketry?  Or I guess leave it as it is if that's intentional.
- You get a _lot_ more upgrade points with all the 1/2/3pt nodes in the early tree.  That might be unbalancing things a little, idk.

**Comment by ec429** [comment]
The 0.625m heatshield (LEO) seems to have disappeared.  Should that be in the same node as the 1m and 1.5m?

**Comment by pap1723** [comment]
@ec429 I will comment on your previous one tomorrow, but yes, the 0.625m heatshield should be in that same node I have a screwed up name for it! Ahhh!! Will fix for tomorrow.

**Comment by NathanKell** [comment]
@leudaimon / @pap1723
1. Concur on Aerobee parts in Start, for sanity.
2. Avionics: Sputnik PS-1 should be in Avionics Prototypes IMO. It doesn't do anything, it's just a heavier sounding rocket unit with more battery life. Explorer 1 and Vanguard 1 (and the 20in XRay which was to be Vanguard 2) should be in Early Avionics IMO--they were created with off-the-shelf 1957 tech. There was no technical reason Explorer 1 could not have been launched in, say, 1956, and its instruments did not appreciably increase in quality/decrease in mass in the next few years. I tend to think it's OK if people reach orbit using 1955-56 tech (or even earlier tbh), because it's really a dead end, it's just for show. (And, tbh, they knew this at the time which is why so many of the proposals kept getting shot down.)
3. Comms. At one point we had a Comm-8 which might be a good idea to bring back. It would be equivalent to the PS-1 antenna but extendable.
4. Concur on tiny solar cell.
5. Electronics: We talked about battery techs. Let's earmark that for these nodes.


Now as to @ec429's comments:
1. I forgot all about the A-9. I'm wondering if we should push the A-4 (and RD-100) back to Start. Or do we think it makes sense to force an early light sounding rocket launch?
2. Comms, per above.
3. Concur on seps. Given their horrific mass ratios, and specific impulses not any different from WWII-vintage solids, I see no real need for the small and medium seps to be in anything but start. Maybe just maybe put the medium in a new solid node with the Aerobee kick motor (the one from Taerobee, not the Tiny Tim) between start and basic solids? Per my engine comments in prior post.
4. Pap and I talked about making the first few points you invest in R&D upgrades not count as much, to slow the player's roll. That's especially vital here, and fast early R&D is not so vital (to non-boring gameplay) now that there are many small nodes early.

**Comment by Theysen** [comment]
From the little playtime I had with the new tree - one thing I came across which put me off was the fairing and interstage fairings and adapter plates missing from the start node. I am pretty sure the tech at the 1950 would have enabled building at least the structural interstage parts. Correct me if that's all intended and a major gameplay mechanic to have fairings and structural parts coming in the later tech tree. (We stock or what? 😅)

**Comment by pap1723** [comment]
@NathanKell 

OK, I have continued our research into the RD-107 variants from our discussion last night. I will present my information and then you can let me know if you think it is worth it, or not, to add any additional variants.

**Current Variants Implemented**
 * RD-107_8D74PS (Used on first 2 Sputnik launches)
 * RD-107_8D74K (Used on Molniya)
 * RD-107MM_8D728 (Used on Molniya-M)
 * RD-117_11D511 (Used on Soyuz-U)
 * RD-107A-14D22 (Used on Soyuz-FG)

After looking at [This Page ](http://www.b14643.de/Spacerockets/Specials/Russian_Rocket_engines/engines.htm)and [This Page](http://www.b14643.de/Spacerockets_1/East_Europe_1/Semyorka/Propulsion/history.htm), I can come up with two additional variants (the other variants all have the same stats as other types).

The first additional variant is the RD-107_8D76 that according to [Gunter's Site](http://space.skyrocket.de/doc_lau_fam/r-7.htm) was only used to launch Sputnik 3. Based on the statistics from the above links, this engine has a slight increase in thrust from 972.3 to 972.8 and ISP from 306 to 310.

The other additional variant is the RD-107_8D74 that was used on the earliest Vostok launches. It had a thrust of 991 and an ISP of 313.

**Proposed Variants**
 * RD-107_8D74PS (Used on first 2 Sputnik launches)
 * RD-107_8D76 (Used on Sputnik 3)
 * RD-107_8D74 (Used on early Vostok launches)
 * RD-107_8D74K (Used on Molniya)
 * RD-107MM_8D728 (Used on Molniya-M)
 * RD-117_11D511 (Used on Soyuz-U)
 * RD-107A-14D22 (Used on Soyuz-FG)

**Proposed Tech Progression**
 * 1956-57 - RD-107_8D74PS
 * 1958 - RD-107_8D76
 * 1959 - RD-107_8D74
 * 1960 - RD-107_8D74K
 * 1964 - RD-107_8D74PS
 * 1973 - RD-117_11D511
 * 2001 - RD-107A_14D22

The RD-108 will go through a similar progression, but has slightly different model variant numbers.

**Comment by NathanKell** [comment]
8D74K was also used for the crewed Vostok launches, AIUI. 8D74, being used for the first Vostok LV variant, was thus used for the Luna launches.

I think we can get by skipping 8D76, and then the comparisons would be:
8D74PS - S-3 - LR43
8D74 - S-3D - first sane LR89
8D74K - MB-3-I (whatever number it was) - next LR89 (for Mercury), early H-1

Do you think 8D76 is worth it?

I had thought there was an uprating during Molniya flights but before Molniya M, but perhaps not.

**Comment by pap1723** [comment]
Alright, major Tree Update. Here are the details:

Thanks to @leudaimon @ec429 @NathanKell @Theysen 
* Added a Tiny ST1 Solar Panel (In RO not included with RP-0)
* Moved FASA Tiny Solar Panel and new RO Tiny Solar Panel to Electronics Satellite Era Blue Sky Node
* Fixed some part name errors causing some parts not to appear in tree (0.625m Heatshield and others)
* Moved J-2 to 1966 since the earliest config unlocks there
* Added a second Nuclear Propulsion tech to Near Future
* Added a Early Solid Rocket tech node
* Moved the SSTU Lander Fuel Tanks to Advanced Capsules Era Materials Science
* Moved all fairings and pieces to the 1-Point Poost-War Materials Science Node. They all have entryCost of 0. This is done because they are not needed for the first launches and this way, the player can unlock only the ones they want to have instead of the whole group of them being available from the start.
* Changed 1956-1957 nodes to be 7 research points each
* Moved Sputnik to Avionics Prototypes
* Added all BDB parts
* Moved A-4 back to start, RD-100 and A-9 available in first Rocketry node
* Moved Taerobee Solid to new Early Solids Node
* Moved Raidernick GCR to new Early Solids Node
* Moved most Taerobee parts to Start node
* Moved the Mk1 Conic Cockpit to the Post-War Flight Tech Node
* Major re-shuffling of engines in the first 6 nodes (SEE BELOW)
* Pushed the 1956-1957 Solid Engine node back to 1958
* Moved the Communotron 16 to the Lunar Range Comms Node
* Moved the Communotron-16-S (Half Range of regular) to the start node
* Moved Balloon Tank up to Satellite Era



Get it from here:
https://github.com/KSP-RO/RP-0/tree/Pap-TechTree

Details on Engines in Nodes:
https://docs.google.com/spreadsheets/d/14k9rJ4UaMWN9s4rNtTkDl_-fMvSj4_OqOw5LlpUpPAE/edit?usp=sharing

**Comment by NathanKell** [comment]
So, first, don't be skeeered by how long the comments to follow are--you've done, and are doing, amazing work, and I for one love it! So, the thoughts:

1. I was a bit confused about the J-2 from what you said, but the chart makes clear. Makes sense to me. I'd put the LR87-LH2 in with the RL10A-3-1.
2. Not sure about shifting the 56/7 solids back to 58. Reasoning?
3. The RD-120 in hydrolox1976--what's that? Wasn't the RD-120 a kerolox engine? You have the RD-0120 (the hydrolox SSME-alike) in 1986...
4. I'd bump the RD-100 back to the start node (it's basically just a retooled A-4), and put the RD-101 along with the A-9, so the 101 and 102 aren't clumped.
5. We might consider moving the Gamma engines back to 1956 so they're a viable counterpart to Vanguard or Juno. They're basically shrunked engines from Black Knight, and that was 56.
6. The RD-0110 actually didn't get used until Molniya-M, I think; the earlier variant is the RD-0107, which we need to add...
7. AJ10-104D was apparently a later mark of AJ10-104; the first 104 flight was in January 1960. I don't think the stats varied significantly however, I'm pretty sure all the 104s had that extended bell. Anyway, that's a reasonable argument for pushing the 104D up a node (and perhaps renaming it just 104) such that it does unlock a year 'earlier' than the first relightable XLR81.
8. I see no reason to keep the AJ10-118 in RP-0 (there's reason in RO itself). The 118D is sooo much better and unlocks at the same time.
9. Altair is definitely more advanced than the Baby Sergeant solids (same tech as the GCR kick motor, really), and those Babies are derivatives of Sergeant/Castor. Per above, would be interested in the reasoning there. For my part, I'd suggest something like early solids with the Aerobee kick and the bigger seps, then a 56 node with Castor, Baby Sergeant (the T17-E2 config), and the GCR kick, and then a 58 node with Altair and the JPL config for the Baby Sergeant. Along with the Pioneer solids and the proc solid. 1959 would have Scout, Algol, etc. Although for gameplay reasons I can see why having a Castor early is problematic--but perhaps fixable with TF.
10. Waxwing is a kick motor, should go in 1966 or 67 in terms of performance IIRC.
11. Wondering about why the RD-253 comes before the UR-500/Proton's uppers.
12. RS-25s should go in hydrolox (SSME, IMO, is hydrolox before it's staged combustion). It's worth considering whether, as is done in the old tree, hydrolox 1980 _requires_ staged combustion to be researched, or no.
13. LR101-NA-15 - I thought that got used as early as the 1960s Deltas? But I may be totally confused here.

That's it for now--again, you're doing awesome stuff, and I'd love to hear your thoughts on those above bits, as I said. They're as much question as statement. :)

**Comment by pap1723** [comment]
> 1. I was a bit confused about the J-2 from what you said, but the chart makes clear. Makes sense to me. I'd put the LR87-LH2 in with the RL10A-3-1.

This is done. It was a mistake on my part as the LR87 had gotten stuck in Orbital.

> 2. Not sure about shifting the 56/7 solids back to 58. Reasoning?

When I had read your previous comment about the nodes, that is what I understood you to mean. It is moot now as the other changes you discussed are below.

> 3. The RD-120 in hydrolox1976--what's that? Wasn't the RD-120 a kerolox engine? You have the RD-0120 (the hydrolox SSME-alike) in 1986.

Yep, typo on my part, now fixed.

> 4. I'd bump the RD-100 back to the start node (it's basically just a retooled A-4), and put the RD-101 along with the A-9, so the 101 and 102 aren't clumped.

Done and I agree with the reasoning.

> 5. We might consider moving the Gamma engines back to 1956 so they're a viable counterpart to Vanguard or Juno. They're basically shrunked engines from Black Knight, and that was 56.

Done, I know nothing about these (sad to say) so I defer to you.

> 6. The RD-0110 actually didn't get used until Molniya-M, I think; the earlier variant is the RD-0107, which we need to add...

I don't know much about Soviet engines, so having to do this kind of research is fun! I agree we should add the RD-0107 as the first variant of the RD-0110 and they will unlock in 1960 and 1964 respectively.

> 7. AJ10-104D was apparently a later mark of AJ10-104; the first 104 flight was in January 1960. I don't think the stats varied significantly however, I'm pretty sure all the 104s had that extended bell. Anyway, that's a reasonable argument for pushing the 104D up a node (and perhaps renaming it just 104) such that it does unlock a year 'earlier' than the first relightable XLR81.

OK, I have renamed it ot the AJ10-104. I really like your thought process on this one. It has been moved into the 1960 node.

> 8. I see no reason to keep the AJ10-118 in RP-0 (there's reason in RO itself). The 118D is sooo much better and unlocks at the same time.

Works for me. I removed the tech unlock for the config, so it will not unlock in the tree.

> 9. Altair is definitely more advanced than the Baby Sergeant solids (same tech as the GCR kick motor, really), and those Babies are derivatives of Sergeant/Castor. Per above, would be interested in the reasoning there. For my part, I'd suggest something like early solids with the Aerobee kick and the bigger seps, then a 56 node with Castor, Baby Sergeant (the T17-E2 config), and the GCR kick, and then a 58 node with Altair and the JPL config for the Baby Sergeant. Along with the Pioneer solids and the proc solid. 1959 would have Scout, Algol, etc. Although for gameplay reasons I can see why having a Castor early is problematic--but perhaps fixable with TF.

I think this makes the solid engines in the beginning of the tree make a lot more sense! I have added a 1956-57 node and arranged the solids how you recommended.

> 10. Waxwing is a kick motor, should go in 1966 or 67 in terms of performance IIRC.

This is completely correct and I actually had the Waxwing part unlocking in Orbital while the engine config was unlocking in solids. OOPS! Moved to 1967.

> 11. Wondering about why the RD-253 comes before the UR-500/Proton's uppers.

I got some bad information from Astronautix about first launch days I am pretty sure. Fixed, they will all unlock in 1966.

> 12. RS-25s should go in hydrolox (SSME, IMO, is hydrolox before it's staged combustion). It's worth considering whether, as is done in the old tree, hydrolox 1980 requires staged combustion to be researched, or no.

Interesting. I "knew" that it was a Hydrolox engine, but I also remember it as a staged engine so that is why I thought it went there. The giant orange tank is symbolic of the Shuttle and is mostly LH2 tank! Hydrolox makes much, much more sense. I have added a 1981 Hydrolox node that requires the previous Staged Combustion node of 1972 to unlock.

> 13. LR101-NA-15 - I thought that got used as early as the 1960s Deltas? But I may be totally confused here.

From what I can tell, it was not used until MA-5 System which was first used on the Atlas I

Thank you very much for the feedback! This is a huge project that I want to make sure is right, so I would rather have a ton of feedback from everyone to make it play the right way. I will upload the changes and add the new RD-0107 later on.

**Comment by NathanKell** [comment]
1. Ah, yeah! I see it now.
2. Whoops, my mistake then! :D
3. Ah, did you mean the 0120? Then it appears twice, in that node and in the next one too.
4. Cool!
5. They didn't actually fly until 1969, but they're 1956-1958 tech, and they'll be entirely unused if they're not in so early a node (and do give an interesting alternative early on). So that's why I support 56 for them. The whole British rocketry program is actually really interesting if you ever want to look into it--the only nation in the history of the world to gain orbital launch capability and then give it up!
6. Cool! Sounds good.
7. Cool!
8. Ok! We might also remove the config entirely (with a -CONFIG[AJ10-118] {} ) because otherwise there will be a ghost option in the UI that one can never unlock.
9. Great!
10. Sounds good!
11. Ah, gotcha.
12. Yep, it's the first US staged combustion engine--we didn't try for kerolox or anything else because we 'knew' that things other than LH2 coked when run fuel rich and we only knew how to make turbopumps that ran fuel rich. The Russians started oxidizer-rich so they moved right on to closed-cycle (that's minimizing the engineering effort involved of course, Russian engine design has been frankly amazing from the start).
13. Ah, ok! AlternateWars verifies it was the -11 verniers on the RS-27s on Delta. Doesn't look like that was downthrusted at all. Heroic Relics has even the -13s still at 1klbf and only the -15s at ~500lbf.

**Comment by PhineasFreak** [comment]
Should we group the early variants (RD-0106, 0107 and 0108) together? They have almost the same stats as far as i can see.

**Comment by pap1723** [comment]
@PhineasFreak I think yes. None of those variants currently exist, so I will just create a single variant of the RD-0107 as the first config inside the RD-0110 global.

**Comment by pap1723** [comment]
Alright, new large update to the Tech Tree here: https://github.com/KSP-RO/RP-0/tree/Pap-TechTree

Thanks to @NathanKell for the great feedback
* New Icons
* Moved LR87-LH2 to Hydrolox
* Moved SSME to Hydrolox
* 1981 Hydrolox now requires 1972 Staged Combustion
* RD-120 moved to correct node, typo fixed
* RD-100 in start, RD-101 to Post War
* Moved all fairings and bases to Start Node
* Gamma Engines in 1956
* Created the RD-0107 engine config for the RD-0110
* RD-0107 unlocks in 1960
* RD-0110 unlocks in 1964
* AJ10-104D was renamed to AJ10-104 and moved to 1960
* Added a 1956-1957 Solid Engine node
* Moved around a lot of solids from the first 4 nodes
* Proton's engines all moved to the correct node

**Comment by NathanKell** [comment]
Sounds great!

**Comment by Bornholio** [comment]
PART: Tank Life support, shows up in TAC LS items if possible instead of Utility if TACLS is installed.
PROBLEM: V2 (A-4) engine not functional. Reinstalled Realplume and RObranch to fix A-4 problem
PART: Proc Tank starts with Tank Type: **Not Found** http://imgur.com/9B3VD94

1st Rocket
Aerobee return, light on fuel, 5.1 science (shores low+recovery)

2nd Rocket
A-4 full V2 configuration, add temp/pressure (i note historically this would have carried cosmic ray sensors (xray/gamma/muon) ) 16 science (shores high, water high, water space)
mixed these up launching sounding rocket medium while waiting for techs. put most cash into R&D points

Part: should procedural avionics have a higher entry cost 2¥, since 1m starting is 6000¥, any reason to use it at this point vs A-4 other than size.

3rd Rocket
A-5, proc tank, with detachable upper w/heatshield, messed up parachute. good for 13science over grasslands and low water.

PART: why is service bay 4000¥&16000¥ neither is in a useful diameter when allowed, 2m is close to usefull but too big, if a 1-1.25m was available that would make more sense. 1.25 would match fixed tanks at same time, 1.0 would match avionics fixed available. A 1.25m FL-T200 tank of same height is 20¥

PART: Toroidal Round 8 cryotank available. (for helium? Insulated for LOX) is this too early?

4th Rocket
A-6 rocket upgraded to A-9 Engine, fixed parachutes, lengthed, RCS added, AJ SRB's boosters added
366km ap Sub orbital hop, no ablator used 4.8 science from vehicle, +something in space ~7.5 total
used funds to start pad upgrade. 

lots more Rockets::::::
Most Sounding rocket contract are stopping being worth anything unless combo with other contracts.
Idled my program to allow research to fill out. Almost no contracts worth doing until orbital is possible
All tech up to the 10pt ones researched. 

PART: FL-T200tank, collider inside mesh putting small Rcs on they disapear under texture.

PART: Battery costs ! never using anything but proceedural.

Started to try and Orbit
Cricket 1, RD103, engine failure, 
Cricket 2, Second stage, sep fail spinning out of control, range safety
Cricket 3, RD103, engine failure, 
Cricket 4, RD103, engine failure, 
Between those and the pad upgrade i've spent a health amount of my mission money for first orbit.

Science at this point
I have 33 in reports and probably <12 in recovered craft  , its 25pts for all 1-3pt tech, +7 for Sat Science, 8.7 remains

I'm going to start again to see if i'm playing sub optimal

**Comment by Bornholio** [comment]
Catastrophe Play through
Build #1, had problems again with A-4 engine not showing dV or TWR, reloaded game, replaced engine on build, works...1246¥ 

PART: Proc Heat Shield, does it have lunar and LEO rated separate?  it had huge Ablator. reduced to 10%, is this item too soon? Probably not unless it is capable of Lunar.

1-47days, return capsule A-4. launched west 88deg, landed in central Florida, 37.9 Science, First Flight Plus firsts, 27797¥

Contracts available, karman, sounding rocket 440kmn 2142¥, xplanes low, crewed Soundbarrier
Trouble spending points in R&D, button locking up. spent 37pts,  all kct upgrades to R&D, 23/yr, 0.5bp build speed. 

Build #2 nearly the same stretched to 20t max tank, 1314¥ cost.

Day 49, refurb complete
Day 64 post war rocketry
Day 80 postwar avionics
Day 96 post war materials
Day 98 #2 on the pad

Launch, shutdown on A-4, 49km AP , made it to flying low water 8.2 science including recovery
Contracts available, karman, sounding rocket 410kmn 2088¥, xplanes low, crewed Soundbarrier

Build #3, A4+Aerobee upper (XASR-1), no attempt at recovery 893¥

Day 99 recondition
day 113 supersonic plane dev, added 2 more plane techs to list without button lockup
Day 136 #3 on the Pad, took karman line and 310km Sounding Rocket

Launch, early A-9 shutdown, but 300km+ap, 384km AP total, got both shores and water space near, water high.  Aerobee fractured on lower atmosphere...hard splash Science 13.9 Science, bank up to 76542¥

Took Sat era Matscie, '56 Orbital Rocketry, purchase Pad Upgrade 16k¥ (137d time), science rate to 33.5/yr
Contracts available, First Sat, SO&return, SR(H-2500km), SR(M-330km), X-Planes(low), First Solar Sat, Sound Barrier

Build #4, change nosecone to parachute, added sep motors and decoupler. 920¥
Took SubOrb&Return and 130km SR(M) contracts
Day 138 Early Rocketry
Day 159 Early SR Engines
Day 175 #4 on the pad

Launched headed south, good spin stabilization at 100km 50deg pitch, full burn on all engined, 850km AP, got some tropics data, but blew up from G loading at 20km, reverted
Lowered AP to 305km, full burn, transmit tropics space data, got flying high tropics, no transmission  time barely survived thermal and g loading. Total 13 science, 73,932¥

Added Sat Era Electronics '56 SR engines and Early Avi&probes. upgraded science research to 0.107pts per day

At this point i have only bought two part upgrades 
Contracts available, First Sat, SO&return, SR(H-5900km), SR(M-250km), X-Planes(low), First Solar Sat, Sound Barrier, Pass karman line Crewed

Build #5, Stubber Comms Plane, 231 days 1200¥
Build #6, Black Iron, 20t, Main A-4 stage, 2stage+sidebooster Aerobee Upper 8335m/s 40day, 994

Day 190 Early Science
Day 216 Launch #6 at pad, forgot to mount on stabilization, A-4 too short run time, rollback

Build#7, Black Iron II, add fins, put spin on Seps, change A-4 to A9, 8723m/s, 42day, 1056

Day 218 Avionics Prototype
Day 246 Early materials Science
Day 259 Launch #7 at pad

Black Iron II had A-9 shutdown early, one of the side booster XASR-1 failed, so boosters ejected on emergency, stage 3 was wobbly and failed to light, seps got final stage clear and lit last engine, 228km AP, blew up on reentry. Success on 170km SR(M) contract, 73330 Funds

Purchased biosample capsule, Geiger tube, micro meteor detector and proc battery 
PART: Biological Sample Capsule, should buying one give price breaks on others?
PART: Z100 Battery, description has wrong values listed for RP0/RO, Cost
PART: Z200 description doesn't apply well, Cost
PART: Z400 Battery, description has wrong values listed for RP0/RO, Cost
PART: TR-2C Stack Separator, needs cost similar to TR-18A
PART: FL-A10 Adapter, Cost, should be low or people will default to Proc Structural
PART: FL-A5 Adapter, Cost, should be low or people will default to Proc Structural

**Comment by NathanKell** [comment]
1. I've never used the service bays myself and have no particular input on them. Whatever y'all think. Ditto batteries.
2. Round 8 should unlock when cryo tanks do, yeah.
3. Sounding rocket contracts per IRC (we'll use the same technique as sat contracts, to require interesting and more powerful designs).
4. No idea about the proc heat shield; AFAIK it only supports one ablator configuration and therefore can't switch between LEO and lunar. If people actually use it, we probably need to clone it so one is LEO-rated and unlocks in the early EDL bits, and one is lunar rated and unlocks when the other lunar shields do. It appears we don't actually change the proc shield's ablator module so I presume it gets whatever DRE gives it, and I have no idea what that's capable of resisting.
5. Bio capsules should have ECMs, but Pap just moved them back to start so it's moot.

The other stuff I'm sure Pap has already responded to and/or dealt with. :)

Thanks so much for the great feedback!

**Comment by PhineasFreak** [comment]
@NathanKell For the PP heat shield point: i have done it for my own purposes and it works great! The only problem is that the PP code for it does not support an output resource ("CharredAblator") and that means the vessel gets much lighter than if you used a normal heat shield.

If someone could take a look and add such a functionality then he/she would be an awesome person! And it would also mean that we can mark it as compatible with RO.

**Comment by NathanKell** [comment]
Wait does it not use a standard ModuleAblator (or DRE's ModuleHeatShield) ?

**Comment by PhineasFreak** [comment]
BIG correction: mistake, it does use it but you cannot define the CharredAblator resource as a resource linked to the Ablator one. You can have both of them defined in the part config but what maxAmount can you set to the CharredAblator one (both of them are set via the stock RESOURCE nodes)?

The amount of the ablative resource is set via the **ProceduralHeatshield** module (more specifically the **ablatorPerArea** parameter). This should probably be updated to include an **outputResource** and **outputResourcePerArea** field to set the name and amount of CharredAblator produced.

**Comment by Bornholio** [comment]
Day 259 Took 1st Sat contract, spent money on Tracking, mission control and R&D
Day 264 New rocketry tech
Day 274 Build 8, black Iron IV, research set on snubotrons, added instruments, eliminated reentry capability.

Day 330 Launch 8, first snub pair one failure, XASR-1 Failed on side boosters, snub fails on last circ booster, 2900km AP, got a bit of science on the way. Sounding High and Medium complete.

can't get tech to select button, tried in and out 12 times, trying to take 1958 orbital rocketry
build 9, Black Iron V, simpler motor setup on upper stage, stretched center A-9 stage
Y2, Day2 Got Sat Era electronics, yeah tiny solar

Day 18 launch 9 main sustainer stage failure, tries 15 times to get to orbit. best attempt was 7500m/s 377km AP, TF stymied the run every time.

Installed FASA.
PART: FASA small solar panel and St1 Tiny should offset the cost of the other, seems silly to make functionally identical parts cost twice.

Bought some parts, mainly the A-6 engine
Build 10, Designed Black iron around A-6 engine and AJ10-27's
Still can't pick any tech nodes

2Y Day 114, Launch 10, easy peasy straight to orbit, I ran a sloppy upper orbit tune, left 1800dV in tanks, from over design for early shutdowns, 754kmx170km, got full TF data for the A-6's in first run
Completed first Sat and Sounding Rocket 460km

Build 11, Iceberg I, a slightly modified B.Iron6, added solar cells (Lots), reduced battery, added new instruments. 6088, Take First Polar Sat, and First Solar Sat, spent 40k on build speed upgrade, 24 k on 150t launchpad

Day 114 mature supersonic flight
Day 160 Finished tracking station (mission control earlier)
Day 208 Material Science finished
Day 219 11 at the Launchpad, good launch into 8.2Mx245km, pulled PE up to 395km using RCS, completed First Polar/Solar and both Sounding Rocket Contracts  Have 107 science i cannot spend (because of tech node pick problem) have 61856 funds

SOLUTION: got this kind note from NK ::Bornholio: It happens when you unlock a node that has parts, the entry cost of which is more than you can afford (so the Purchase All Parts button is unclickable). You have to then select a purchased node where you *can* afford to purchase all the parts, and hover over the Purchase All Parts button, and that gives you your highlight back.::

IDEA: Icon for ED&Landing tech could be a probe (antenna ticking out) at angle with HS & shockwave
IDEA: Icon for ED&Landing tech could be a mercury pod at angle with HS & shockwave 

PART: CR2 Is this supposed to represent the Ramjet carried by X-15A-2 in '67?  Engine should have matching shock cone include in Node
	Ramjet should be http://www.okieboat.com/Talos%20missile.html Engine, by 1958 enough of these made and used for all kinds of applications      -Imaginary parts!  how about one from https://pdfs.semanticscholar.org/9330/17c8df43bfe6978ac0707595e53cb0c94829.pdf page 2 
http://www.okieboat.com/Ramjet%20history.html  I'll open an issue in RO to work on real ramjet models.

PART: Jets, Avron and J57 and Derwent V should have a circular intake that matches diameter. This is easy to ignore with tweakscale...
PART: Shock Cone Intake, to big for available jets

Build 12, small sounding rocket with bio samples for Air sampling in nearby biomes

**Comment by PhineasFreak** [comment]
@pap1723 A small addition to the RP-0 part patching: the ability to automatically set the entry costs to 0 for any parts that are placed in the starting node. This can always happen if you add a new mod (or if a part is moved) and it saves some time hacking the save file to mark them as unlocked.

(I was watching the TTT streams and realized that was somewhat annoying getting new parts that need funds to unlock).

Or is it taken care by the new "identicalPart" tag?

**Comment by pap1723** [comment]
@PhineasFreak the functionality of KSP takes care of that (from what I understand). The only reason that has happened is with us moving parts into the start node in the middle of a game. 

If you start a game from scratch, all the Start node parts are automatically unlocked for free.

**Comment by Bornholio** [comment]
US biological sample container is 2000$ in starting node
Fasa Fuel tank is 3000$ in post war material tech node
Pressmat-Hot-Xray US module, do you want it here also 1000$ Early if intended to be limited by GM tube.
Geiger Muller tube is supposed to represent the full spectrum of (Xray-Solar Cosmic ray, gamma ray and GCR) along with magnetic field instrumentation? Antimatter detected in 1919 with geiger in balloon, 1946, first A-4 flights had Cosmic ray detectors (assume GM Tube or other Gas discharge type) They are easier to transmit data from via analog, as clicks are the basic output. temp and baromter either need tone control or click frequency to output.

Kevlar for chutes is available too early, 1966 earliest, Oldest I could find was Mil study for parachutes made in 1974 (ribbon chutes for Nuclear bombs) Apollo chutes are Nylon
 
add Terylene chutes... Supersonic Ribbon Parachutes(1980's)

okay done blathering

**Comment by NathanKell** [comment]
On chutes, I'm not aware of tech-gating chute materials. @stupidchris is that possible?

On GM tubes, hmm. That's a good point. Not sure what we can do with it though...

**Comment by Bornholio** [comment]
Probably ignore GM's that early and assume that greater detail and better instruments available to Van Allen's time period is placed ok for game balance (you get it pretty fast).

**Comment by Bornholio** [comment]
https://www.dropbox.com/s/ysjctyzjzy06oqb/HIAD%20Icon.png?dl=0  Hypersonic Decel
https://www.dropbox.com/s/5tmk2wom6w0sik9/Supersonic_Decelerator%20ICON.png?dl=0  Supersonic Decel

https://www.dropbox.com/s/5tmk2wom6w0sik9/Supersonic_Decelerator%20ICON.png?dl=0 List of NTRs
Includes stock SXT, Vens, Kerbal Atomics, and Porks Atomic Age.  Has initial entries for early RD-4XX series but too little information is available to fill in the details or back calculate values. Lastly the existing RO configs are listed for comparison.  Still no NASA current notional ones added (Constellation, SNRE, RFNTR or Arcjet)
